### DISCUSSION TITLE
ScriptReference/Physics2D-queriesStartInColliders

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#597] [Notes]
# ScriptReference/Physics2D-queriesStartInColliders

https://docs.unity3d.com/ScriptReference/Physics2D-queriesStartInColliders.html

**Comment by rhys-vdw** [comment]
Also affects [Physics2D.BoxCast](https://docs.unity3d.com/ScriptReference/Physics2D.BoxCast.html).
